docs/conf.py

```diff
@@ -361,7 +361,7 @@ def _get_rst_filepath_from_path(filepath: str):
 
 # == Extensions configuration ==================================================
 
-# -- Options for sphinxcontrib.jinjac ------------------------------------------
+# -- Options for sphinx_jinjac ------------------------------------------
```

Comment:
```suggestion
# -- Options for sphinx_jinja ------------------------------------------
```

typo?

Reply:
I guess.. the `c` was there before but lets remove it